docs/digger/installation/macos-variables.asciidoc

```diff
@@ -1,17 +1,15 @@
----
```

Comment:
@laurafitzgerald what is this and why did i remove it? :)

Reply:
```
----
-layout: post
-section: docs
-title: Digger Ansible macOS variables
----
```

@laurafitzgerald

Reply:
https://jekyllrb.com/docs/frontmatter/
@laurafitzgerald we don't need this here i presume, it is not present in other docs (with exception of masters)

resolved